homeassistant/components/zwave_me/manifest.json

```diff
@@ -0,0 +1,17 @@
+{
+  "domain": "zwave_me",
+  "name": "Z-Wave.Me",
+  "documentation": "https://github.com/Z-Wave-Me/ha-core",
+  "iot_class": "local_push",
+  "requirements": [
+    "zwave_me_ws==0.1.21",
```

Comment:
The GitHub repo for the library doesn't seem to have been updated with the latest released code. I'd like to view the source of the latest release. Also please tag GitHub releases whenever releasing to PyPI to make it easier to track changes between releases.

Thanks!

Reply:
@lawfulchaos https://github.com/Z-Wave-Me/zwave-me-ws/commits/main shown no commits for lat two weeks. Looks like the release tag is added without pushing new sources.

Reply:
commited and retagged

Reply:
fixed

Reply:
The `close_ws` method doesn't wait for the executor tasks to finish. Should it do that?